pkgs/development/libraries/ncurses/default.nix

```diff
@@ -139,6 +139,11 @@ stdenv.mkDerivation (finalAttrs: {
       done
     done
 
+    # add pkg-config aliases for libraries that are built-in to libncursesw
+    for library in tinfo tic; do
+      ln -svf ncursesw.pc $dev/lib/pkgconfig/$library.pc
+    done
+
```

Comment:
It might be slightly cleaner to create `tinfow.pc -> ncursesw.pc`, `tinfo.pc -> ncurses.pc`. Otherwise LGTM.

Reply:
Checking https://packages.debian.org/bookworm/amd64/libncurses-dev/filelist which builds in split mode, it doesn't seem like tinfo/tic ever have a wide version. I guess I could symlink to ncurses.pc always? WDYT?

Reply:
AFAIU upstream build always tries to share the same chosen suffix across libraries: https://github.com/mirror/ncurses/blob/master/misc/Makefile.in#L138

Some systems (like `gentoo`) do install wide (`tinfow`) and non-wide (`tinfo`) ABI versions in a single package and there mixing `tinfo` and `tinfow` are disastrous: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=63d949769e25176de0d879cbef2b7cd80348ad0f

It does not matter much in practice for `nixpkgs` as it builds only one version of `ncurses` ABI in a single derivation (and `nixpkgs` then adds symlinks to emulate other ABIs but dos not really provide non-wide versions). Up to you.

Reply:
Pushed a new version of this PR:

$ ls -l /nix/store/gpildrn5fywjjaw3yslx9dp6ryzcnyi4-ncurses-6.4-dev/lib/pkgconfig
total 28K
dr-xr-xr-x  2 root root 4.0K Jan  1  1970 .
dr-xr-xr-x  3 root root 4.0K Jan  1  1970 ..
lrwxrwxrwx  9 root root    8 Jan  1  1970 form.pc -> formw.pc
-r--r--r--  2 root root  642 Jan  1  1970 formw.pc
lrwxrwxrwx  9 root root    8 Jan  1  1970 menu.pc -> menuw.pc
-r--r--r--  2 root root  642 Jan  1  1970 menuw.pc
lrwxrwxrwx  9 root root   13 Jan  1  1970 ncurses++.pc -> ncurses++w.pc
lrwxrwxrwx 13 root root   11 Jan  1  1970 ncurses.pc -> ncursesw.pc
-r--r--r--  2 root root  674 Jan  1  1970 ncurses++w.pc
-r--r--r--  2 root root  633 Jan  1  1970 ncursesw.pc
lrwxrwxrwx  9 root root    9 Jan  1  1970 panel.pc -> panelw.pc
-r--r--r--  2 root root  644 Jan  1  1970 panelw.pc
lrwxrwxrwx  3 root root   10 Jan  1  1970 tic.pc -> ncurses.pc
lrwxrwxrwx 13 root root   11 Jan  1  1970 ticw.pc -> ncursesw.pc
lrwxrwxrwx  3 root root   10 Jan  1  1970 tinfo.pc -> ncurses.pc
lrwxrwxrwx 13 root root   11 Jan  1  1970 tinfow.pc -> ncursesw.pc